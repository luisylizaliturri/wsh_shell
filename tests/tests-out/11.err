wsh: command not found: echo
