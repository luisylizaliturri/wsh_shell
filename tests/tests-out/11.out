wsh> Executing command: export
wsh> wsh> 